[EDU-2771] docs: improve old functions section in Rules Engine (#418)
aziontech/docs: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx

---

## Edge certificate

> Se você quiser alterar seu domínio, você pode registrar seu próprio certificado TLS (X.509) no [Real-Time Manager](https://manager.azion.com/) sem custo adicional. Para utilização desse recurso, você precisará comprovar a propriedade do domínio.

Author: ```suggestion
Se você quiser usar um domínio próprio, você pode registrar seu próprio certificado TLS (X.509) e chave privada em formato [ASCII PEM](https://www.ssl.com/guide/pem-der-crt-and-cer-x-509-encodings-and-conversions/) no [Real-Time Manager](https://manager.azion.com/) sem custo adicional. A chave privada não pode estar protegida por *passphrase*.
```

> Se você quiser usar um domínio próprio, você pode registrar seu próprio certificado TLS (X.509) e chave privada em formato [ASCII PEM](https://www.ssl.com/guide/pem-der-crt-and-cer-x-509-encodings-and-conversions/) no [Real-Time Manager](https://manager.azion.com/) sem custo adicional. A chave privada não pode estar protegida por *passphrase*.

Author: ```suggestion
Se você deseja usar um domínio próprio, você pode registrar seu próprio certificado TLS (X.509) e chave privada em formato [ASCII PEM](https://www.ssl.com/guide/pem-der-crt-and-cer-x-509-encodings-and-conversions/) no [Real-Time Manager](https://manager.azion.com/) sem custo adicional. A chave privada não pode estar protegida por *passphrase*.
```

> Para adicionar seu certificado próprio, você precisará do par Certificado (X.509) e da respectiva chave privada, ambos no formato [ASCII PEM](https://www.google.com.br/search?q=Como+converter+um+certificado+PFX+para+PEM&cad=h). A chave privada não pode estar protegida por *passphrase*.

Reviewer: é uma info diferente da que ta em EN

Author: Verdade! Vou corrigir isso 😨

> Para adicionar seu certificado próprio, você precisará do par Certificado (X.509) e da respectiva chave privada, ambos no formato [ASCII PEM](https://www.google.com.br/search?q=Como+converter+um+certificado+PFX+para+PEM&cad=h). A chave privada não pode estar protegida por *passphrase*.

Author: ```suggestion
Quando seu certificado é registrado com a Azion, edge applications com o certificado usam a extensão Server Name Indication (SNI) do protocolo TLS. Verifique a [lista da navegadores com suporte a SNI](https://caniuse.com/#feat=sni).
```

> Quando seu certificado é registrado com a Azion, edge applications com o certificado usam a extensão Server Name Indication (SNI) do protocolo TLS. Verifique a [lista da navegadores com suporte a SNI](https://caniuse.com/#feat=sni).

Author: ```suggestion
Quando seu certificado é registrado com a Azion, domínios com o certificado usam a extensão Server Name Indication (SNI) do protocolo TLS. Verifique a [lista da navegadores com suporte a SNI](https://caniuse.com/#feat=sni).
```
